### DISCUSSION TITLE
new suggestions

### DISCUSSION THREAD

**Discussion by aemzayn** [tomickigrzegorz/leaflet-examples#71] [Announcements]
hi dude I like your works!

I have few suggestions
1. Polygon with gradient fill color
2. Add text to leaflet (for place labeling)

**Comment by tomickigrzegorz** [comment]
Thanks for your suggestions.
I implemented the first idea [polygon-with-gradien](https://tomickigrzegorz.github.io/leaflet-examples/#70.polygon-with-gradien)
As for the second one, I have no idea what you mean, maybe you can describe what you had in mind?

**Reply by aemzayn** [reply]
For example, I have a group of markers and these markers represent equipment in real life, these markers are grouped in a polygon as a station. The problem is there are a lot of these stations and these stations need to be named and the name has to be always visible (not like popup), I give an example of how this would look like

![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/44734533/2db6b6b1-a4b8-4752-b6a5-3cc79ee5f20a)

**Reply by tomickigrzegorz** [reply]
Finding an example of how to add text next to a marker can be found within 5 seconds of a google search 😀
But I'll see if I find time, maybe I'll add this example to the examples as well.
Thanks

PS. Hint, you can use "bindTooltip" and permanently enable it under each marker.
